Update SandboxieTrace.md (#49)
DavidXanatos/sandboxie-docs: Content/SandboxieTrace.md

The settings **PipeTrace**, **IpcTrace** and **GuiTrace** are more relevant to the discussion in this page. **FileTrace** and **KeyTrace** will usually not be able to provide insight as to why a sandboxed program is malfunctioning.

Thus, typically you enable the trace by making this change in [Sandboxie Ini](SandboxieIni.md):
```
   [GlobalSettings]
   IpcTrace=ad
   PipeTrace=ad
   GuiTrace=ad
```

Then use Sandboxie to reload the configuration:
* **Configure** menu -> **Reload Configuration** on Sandboxie Classic
* **Options** menu -> **Reload ini file** on Sandboxie Plus

Trace options can be set on a per box basis such that only the boxes you need will generate trace logs.

You can also adjust the buffer size by adding ```TraceBufferPages=2560``` that will increase it tenfold.

### Review the Trace for **NetFwTrace**, **IpcTrace** and **PipeTrace**

> Since version 0.9.0 / 5.51.0, it was added a new option `NetFwTrace=*` to trace the actions of the firewall components. Please note that the driver only logs the kernel debug output, hence you can use [DbgView.exe](https://docs.microsoft.com/it-it/sysinternals/downloads/debugview).

Reviewer: ```suggestion
Since version 0.9.0 / 5.51.0, a new option `NetFwTrace=*` was added to trace the actions of the firewall components. Please note that the driver only logs to the kernel debug output, which you can view with [DbgView.exe](https://docs.microsoft.com/en-us/sysinternals/downloads/debugview).

```

Reviewer: changed link to the english site
